components/NavigationDrawer/src/MDCBottomDrawerViewController.h

```diff
@@ -0,0 +1,46 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@protocol MDCBottomDrawerHeader;
+
+/**
+ View controller for containing a Google Material bottom drawer.
+ */
+@interface MDCBottomDrawerViewController : UIViewController
+
+/**
+ The main content displayed by the drawer.
+ Its height is determined by the returned preferred content size.
+ */
+@property(nonatomic, nullable) UIViewController *mainContentViewController;
```

Comment:
Naming suggestion for consistency with other APIs: `contentViewController`.

https://github.com/material-components/material-components-ios/blob/develop/components/AppBar/src/MDCAppBarContainerViewController.h#L62

Reply:
done